enterprise/jmx-monitoring/src/main/java/io/crate/plugin/CrateMonitor.java

```diff
@@ -62,7 +63,7 @@ public CrateMonitor(JobsLogs jobsLogs,
                         ClusterService clusterService,
                         ThreadPool threadPool,
                         CrateCircuitBreakerService breakerService) {
-        logger = Loggers.getLogger(CrateMonitor.class, settings);
+        logger = Loggers.getLogger(CrateMonitor.class, Node.NODE_NAME_SETTING.get(settings));
```

Comment:
isn't the node name now logged by default due to log4j pattern change and logger-with-nodename initialization?